atomic_reactor/schemas/config.json

```diff
@@ -836,6 +836,11 @@
         },
         "additionalProperties": false,
         "required": ["url"]
+    },
+    "builder_ca_bundle": {
+        "description": "The path to the ca-bundle certificate inside the buildroot.",
+        "type": "string",
+        "default": "/etc/pki/ca-trust/extracted/pem/tls-ca-bundle.pem"
```

Comment:
Where is this default value used? I cannot see it in the code

Reply:
The call to `get_builder_ca_bundle`

https://github.com/containerbuildsystem/atomic-reactor/pull/1597/files#diff-65b2ed390ee23c54fcfe0ca79c38e0068d64e05f5b554e5a113f5c4a80546fc7R123

Reply:
I see no fallback there, so it will fail (or return None), but it will NOT use `/etc/pki/ca-trust/extracted/pem/tls-ca-bundle.pem` as a default path when it's not configured.